bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/NibeGW/NibeGW.ino

```diff
@@ -66,15 +68,21 @@
 
 // ######### INCLUDES #######################
 
+#ifdef TRANSPORT_ETH_ENC28J60
+#include <UIPEthernet.h>
+#else
 #include <SPI.h>
 #include <Ethernet.h>
+#endif
 
 #ifdef PRODINO_BOARD
 #include "KmpDinoEthernet.h"
 #include "KMPCommon.h"
 #endif
 
+#ifndef TRANSPORT_ETH_ENC28J60
 #include <EthernetUdp.h>
```

Comment:
Could this be moved to earlier TRANSPORT_ETH_ENC28J60

Reply:
Moved.